src/PrestaShopBundle/Form/Admin/Sell/Product/Options/ProductAttachmentsType.php

```diff
@@ -0,0 +1,87 @@
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Options;
+
+use PrestaShopBundle\Form\Admin\Type\IconButtonType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ProductAttachmentsType extends TranslatorAwareType
+{
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        UrlGeneratorInterface $urlGenerator
+    ) {
+        parent::__construct($translator, $locales);
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('attached_files', CollectionType::class, [
+                'label' => false,
+                'entry_type' => AttachedFileType::class,
+                'allow_add' => true,
+                'allow_delete' => true,
+                'prototype_name' => '__ATTACHMENT_INDEX__',
+            ])
+            ->add('add_attachment_btn', IconButtonType::class, [
+                'label' => $this->trans('Add new file', 'Admin.Catalog.Feature'),
+                'icon' => 'add_circle',
+                'type' => 'link',
+                'attr' => [
+                    'data-success-create-message' => sprintf('%s %s',
+                        $this->trans(
+                            'Attachment was successfully created and added to the selection.',
+                            'Admin.Catalog.Feature'
+                        ),
+                        $this->trans(
+                            'This window is gonna close automatically.',
+                            'Admin.Glocal'
```

Comment:
Hi @jolelievre, is this wording a notification? Where does it appear? I would like to make sure "Admin.Glo**b**al" (there's a typo in your PR) is the right domain.

Reply:
For now, it is only displayed in this feature you can see a preview in this video https://drive.google.com/file/d/1-TWWWP5weZGG6wqjnqkSu-tyLrLH99_f/view

The message is displayed after the new file has been uploaded (in a modal), then the modal is automatically closed. So the wording `Attachment was successfully created and added to the selection.` is definitely related to product page so its namespace should logically be `Admin.Catalog.Feature`

But for `This window will close automatically.` I thought it was more generic and could potentially be used in other context anywhere in the BO.